web/src/routes/EventsPage.tsx

```diff
@@ -96,6 +97,16 @@ const EventsPage = ({ cityModel, languages, pathname, languageCode, cityCode }:
     )
   }
 
+  const downloadEventAsIcsFile = (event: EventModel) => {
+    const blob = new Blob([mapToICalFormat(event)], { type: 'text/calendar;charset=utf-8' })
+    const linkEl = document.createElement('a')
+    linkEl.href = window.URL.createObjectURL(blob)
+    linkEl.setAttribute('download', 'event.ics')
+    document.body.appendChild(linkEl)
+    linkEl.click()
+    document.body.removeChild(linkEl)
+  }
```

Comment:
Do you think this would be possible as well? https://stackoverflow.com/a/56763856/9875582

Reply:
Unfortunately this doesn't work with the blob we create